src/components/Column/Column.test.tsx

```diff
@@ -35,6 +36,36 @@ describe("<Column />", () => {
     fireEvent.blur(renderedEditableLabel);
 
     // Assert
-    expect(mockedUseMutationCall).toBeCalled();
+    expect(mockedUseMutationCall).toBeCalledTimes(1);
+    expect(mockedUseMutationCall).toBeCalledWith({
+      variables: { name: changedMockedValue },
+    });
+  });
+
+  test("passed appropriate props - it should render given ticket with title", () => {
+    // Arrange
+    const mockedColumnId = 2;
+    const mockedColumn = {
+      id: mockedColumnId,
+      name: "mocked column name",
+      order: 0,
+      tickets: [],
+    };
+    const mockedChangedValue = "mocked changed value";
+
+    render(<Column column={mockedColumn} />);
+    const addButton = screen.getByRole("button");
+    fireEvent.click(addButton);
+    const editableTicket = screen.getByLabelText("editable-ticket");
+    fireEvent.change(editableTicket, { target: { value: mockedChangedValue } });
+
+    // Act
+    fireEvent.keyDown(editableTicket, { key: "Enter" });
+
+    // Assert
+    expect(mockedUseMutationCall).toBeCalledTimes(2);
```

Comment:
line 39 and this 66 line, I am not sure if it is common to test this way. LMK if know better practice.

Reply:
You should be able to run `mockedUseMutationCall.clear()` on `afterEach`. The mock function cleans up its usage data so that you can reset the counts of calls of the function, then you won't need to check `toBeCalledTimes` eventually.

docs: https://jestjs.io/docs/mock-function-api#mockfnmockclear